kolibri/plugins/learn/assets/src/views/LibraryPage/AccessibilityResourceCard.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+
+  <div>
+    <KCard
+      :to="to"
+      :headingLevel="headingLevel"
+      layout="horizontal"
+      thumbnailDisplay="large"
+      :title="contentNode.title"
+      :thumbnailSrc="thumbnailSrc"
+      :thumbnailScaleType="thumbnailScaleType"
+      :thumbnailAlign="thumbnailAlign"
+      :style="{ height: '172px',width: '500px', margin: '16px 0 16px 0' }"
+    >
+      <template #thumbnailPlaceholder>
```

Comment:
I don't think we should be showing folder icon on resource cards - is this work in progress?

Reply:
Yes